packages/core/src/mesh/SkinnedMeshRenderer.ts

```diff
@@ -119,27 +119,50 @@ export class SkinnedMeshRenderer extends MeshRenderer {
   /**
    * @internal
    */
-  override update(): void {
-    const skin = this._skin;
-    if (skin?.bones.length > 0) {
-      skin._updateSkinMatrices(this);
+  override _updateTransformShaderData(context: RenderContext, onlyMVP: boolean, batched: boolean): void {
+    const worldMatrix = (this.skin?.rootBone ?? this.entity).transform.worldMatrix;
+    if (onlyMVP) {
+      this._updateProjectionRelatedShaderData(context, worldMatrix, batched);
+      return;
     }
+    this._updateWorldViewRelatedShaderData(context, worldMatrix, batched);
+  }
```

Comment:
const worldMatrix = (this.skin?.rootBone ?? this.entity).transform.worldMatrix;
    if (onlyMVP) {
      this._updateProjectionRelatedShaderData(context, worldMatrix, batched);
    } else {
      this._updateWorldViewRelatedShaderData(context, worldMatrix, batched);
    }

Reply:
done